uniflow/op/model/model_server.py

```diff
@@ -4,6 +4,10 @@
 
 from typing import Any, Dict, List
 
+from functools import partial
+
+import re
+
```

Comment:
You import order is violating the python standard. Please follow instruction [here](https://www.notion.so/goldpiggy/Python-Linter-Setup-30fb3b81f0904af889832e4c697c5ec9?pvs=4) to properly setup your black, isort in vscode IDE.

Reply:
Just fixed the order by following the instruction